shacl/register.shacl.ttl

```diff
@@ -0,0 +1,198 @@
+	sh:and (
+		[ sh:targetClass schema:Dataset ; ]
+		[ sh:not ( [ sh:targetClass schema:WebPage ; ]  ) ]	
+	);
+	sh:nodeKind sh:IRI ;
+	# required properties
+	sh:property reg:SchemaNameProperty, reg:SchemaDescriptionProperty, reg:SchemaCreatorProperty, reg:SchemaLicenseProperty, reg:SchemaDistributionProperty ;
+	# recommended properties
+
+#	TODO
+#	sh:property reg:SchemaPublisherProperty, reg:SchemaDateCreatedProperty, reg:SchemaDatePublishedProperty, reg:SchemaDateModifiedProperty, reg:SchemaVersionProperty, reg:SchemaManEntityOfPageProperty, reg:SchemaInLanguageProperty, reg:SchemaIsBasedOnUrlProperty, reg:SchemaCitationProperty, reg:SchemaGenreProperty, reg:SchemaKeywordsProperty, reg:SchemaSpatialCoverageProperty, reg:SchemaTemporalCoverageProperty, reg:SchemaIncludedInDataCatalogProperty, reg:SchemaAlternateNameProperty ;
+.
+
+reg:DistributionShape
+	a sh:NodeShape ;
+	sh:targetClass schema:DataDownload ;
+	sh:property reg:DistributionContentUrlProperty, reg:DistributionEncodingFormatProperty ;
+.
+
+reg:CreatorShape
+	a sh:NodeShape ;
+	sh:targetClass schema:Organization ;
+	sh:property reg:CreatorNameProperty ;
+.
+
+reg:CreatorNameProperty a sh:PropertyShape ;
+	sh:minCount 1 ;
+	sh:path schema:name ;
+.
+
+#
+# Property shapes
+#
+
+reg:DistributionContentUrlProperty a sh:PropertyShape ;
+	rdfs:comment "The URL where the distribution can be directly accessed."@en ;
+	sh:minCount 1 ;
+	sh:path schema:contentUrl ;
+	sh:message "Een dataset distributie dient een content URL te bevatten"@nl, "A dataset distribution should contain an content URL"@en ;
+.
+
+reg:DistributionEncodingFormatProperty a sh:PropertyShape ;
+	rdfs:comment "The distribution's MIME format, for example application/sparql-query for a SPARL endpoint or application/ld+json for a data dump serialized as JSON-LD." ;
+	sh:minCount 1 ;
+	sh:path schema:encodingFormat ;
+	sh:message "Een dataset distributie dient een encoding format te bevatten"@nl, "A dataset distribution should contain an encoding format"@en ;
+.
+
+reg:DatatalogProperty a sh:PropertyShape ;
+	sh:class schema:Dataset ;
+	sh:minCount 1 ;
+	sh:node schema:DatasetShape ;
+	sh:path schema:dataset ;
+	sh:message "Een datasetcatalogus dient minimaal een dataset te bevatten"@nl, "A datasetcatalog should contain at least one dataset"@en .
+
+reg:SchemaDatasetRequiredProperties a sh:PropertyGroup ;
+	rdfs:comment "Verplichte properties van een datasetbeschrijving"@nl
+.
+
+reg:SchemaDatasetRecommendedProperties a sh:PropertyGroup ;
+	rdfs:comment "Aangeraden properties van een datasetbeschrijving"@nl
+.
+
+reg:SchemaNameProperty a sh:PropertyShape ;
+	rdfs:comment "Naam van de dataset"@nl ;
+	sh:path schema:name ;
+	sh:minCount 1 ;
+	sh:datatype xsd:string ;
+	sh:message "Een datasetbeschrijving dient een naam te bevatten"@nl, "A dataset description should have a name"@en ;
+	sh:group reg:SchemaDatasetRequiredProperties
+.
+
+reg:SchemaDescriptionProperty a sh:PropertyShape ;
```

Comment:
You made description required again, which makes one of the test fail (and it should).